snapcraft_parser/main.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python3
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+snapcraft-parser
+
+Usage:
+  snapcraft-parser [options]
+
+Options:
+  -h --help                             show this help message and exit
+  -p --print                            print the master parts list
+  -v --version                          show program version and exit
+  -d --debug                            print debug information while executing
+                                        (including backtraces)
+
+
+For more help, visit the documentation:
+http://developer.ubuntu.com/snappy/snapcraft
```

Comment:
Removed since this isn't really a script the average snapcraft user cares about.